src/main/java/com/internet/shop/model/ShoppingCart.java

```diff
@@ -0,0 +1,58 @@
+package com.internet.shop.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class ShoppingCart {
+    private Long id;
+    private List<Product> products;
+    private Long userId;
+
+    public ShoppingCart(List<Product> products, Long userId) {
+        this.products = products;
```

Comment:
same - we will not receive products from outside

Reply:
done